src/train_cmd.cpp

```diff
@@ -959,6 +1024,83 @@ static CommandCost CheckNewTrain(Train *original_dst, Train *dst, Train *origina
 	return_cmd_error(STR_ERROR_TOO_MANY_VEHICLES_IN_GAME);
 }
 
+/**
+ * Check if a train can be placed in a depot tile.
+ * @param train The train.
+ * @param tile The tile to check whether it is possible to place the train.
+ * @return whether it found a depot tile to place the train.
```

Comment:
```suggestion
 * @return True if we found a depot tile in which to place the train, else false.
```

Reply:
I will keep it with "@return Whether". This is also used in other comments.